### PR TITLE
Docs: adds external Alertmanager config

### PR DESCRIPTION
**Pull request by brendamuir**
Updates the steps for adding an external Alertmanager per v9.2. 
Adds a deprecation note for the URL configuration.

### CODE REVIEW

**Comment by brendamuir** [comment]
@grobinson-grafana does this cover all the config steps? or anything else we need to call out on that page (gotchas for adding an alertmanager...)

**Review by grobinson-grafana — APPROVED** [review]

File: docs/sources/alerting/set-up/configure-alertmanager/index.md [hunk 1]
```diff
@@ -8,29 +8,29 @@ keywords:
...
 ---
 
-# Configure Alertmanager
+# Add an external Alertmanager
 
-Configure Alertmanager from Grafana Alerting to group and manage alert rules, adding a layer of orchestration on top of your external alerting engine.
+Set up Grafana to use an external Alertmanager as a single Alertmanager to receive all of your alerts. This external Alertmanager can then be configured and administered from within Grafana itself.
 
-## Add a new external Alertmanager
+Once you have added the Alertmanager, you can use the Grafana Alerting UI to manage silences, contact points as well as notification policies. A drop-down option in these pages allows you to switch between alertmanagers.
```
[the remark this answers is not in the document]
Comment (brendamuir) [reply docs/sources/alerting/set-up/configure-alertmanager/index.md]: done-thanks

File: docs/sources/alerting/set-up/configure-alertmanager/index.md [hunk 2]
```diff
@@ -8,29 +8,29 @@ keywords:
...
 
-1. In the Grafana menu, click the Alerting (bell) icon to open the Alerting page listing existing alerts.
-2. Click **Admin** and then scroll down to the External Alertmanager section.
-3. Click **Add Alertmanager** and a modal opens.
-4. Add the URL and the port for the external Alertmanager. You do not need to specify the path suffix, for example, `/api/v(1|2)/alerts`. Grafana automatically adds this.
+**Note:**
+Starting with release 9.2, the URL configuration of external alertmanagers from the Admin tab on the Alerting page is deprecated. It will be removed in a future release.
```
[the remark this answers is not in the document]
Comment (brendamuir) [reply docs/sources/alerting/set-up/configure-alertmanager/index.md]: done thanks!

**Comment by jaygridley** [comment]
Hello, the changes introduced in this PR are not correct. Adding external Alertmanager does not allow Grafana UI to configure its contact points and notification policies. Those are read-only. Only muting/silences are configurable.

**Pull request merged**